src/components/dialogmote/FritekstSeksjonWithAlert.tsx

```diff
@@ -0,0 +1,68 @@
+import { FlexColumn, FlexRow, PaddingSize } from "../Layout";
+import { Field } from "react-final-form";
+import React from "react";
+import styled from "styled-components";
+import { TrackedKnapp } from "@/components/buttons/TrackedKnapp";
+import { AlertstripeFullbredde } from "@/components/AlertstripeFullbredde";
+import Fritekst from "@/components/Fritekst";
+
+const texts = {
+  forhandsvisning: "Forhåndsvisning",
+};
+
+const FritekstWrapper = styled.div`
+  margin-bottom: 4em;
+`;
+
+interface FritekstSeksjonWithAlertProps {
+  fieldName: string;
+  label: string;
+  handlePreviewClick: () => void;
+  maxLength: number;
+  alertText: string;
+}
+
+const FritekstSeksjonWithAlert = ({
```

Comment:
Kan vi unngå å duplisere FritekstSeksjon her, f.eks. ved å utvide den med mulighet for å vise alert?